pkg/auth/auth.go

```diff
@@ -97,6 +97,51 @@ func (c *Client) GetSession(ctx context.Context) (*session.Token, error) {
 	return tok, nil
 }
 
+// GetSessions returns all session tokens. It attempts to refresh any
+// expired tokens, and returns an error if any tokens are unable to be
+// refreshed. If there are errors, it still returns all tokens, some of which
+// may be expired.
+func (c *Client) GetSessions(ctx context.Context) ([]*session.Token, error) {
+	tokens, err := c.store.ReadTokens(c.issuer, c.clientID)
+	if errors.Is(err, os.ErrNotExist) {
+		return nil, ErrNotLoggedIn
+	} else if err != nil {
+		return nil, err
+	}
+
+	var refreshErrors []error
+	for idx, tok := range tokens {
+		if !tok.Valid() {
+			tokens[idx], err = c.refresh(ctx, tok)
+			refreshErrors = append(refreshErrors, err)
+		}
+	}
+
+	return tokens, errors.Join(refreshErrors...)
```

Comment:
The idea behind this API is to allow the caller to still display useful into even if error. For example, you have 3 tokens, 2 work and one failed to refresh. The caller wants to show what orgs are available. In that case, it can display:

* org 1
* org 2
* org 3 (requires re-login)

instead of just erroring out and requiring full login no matter what.

returning a slice of tokens and a slice or errors would be very un-go'ish. Another alternative is to return something like:

```golang
type TokenSource interface {
	Token() (*Token, error)
}
```

(this interface is actually part of oauth2 package)